src/features/transfer/TransferTokenForm.tsx

```diff
@@ -126,22 +139,52 @@ function ChainSelectSection({ isReview }: { isReview: boolean }) {
     return getNumRoutesWithSelectedChain(warpCore, values.destination, false);
   }, [values.destination, warpCore]);
 
+  const handleGetTokensRoute = (origin: string, destination: string) => {
```

Comment:
If a function has `get` near the beginning of it's name, that generally implies it's a pure function (i.e. no side-effects). But this does have side-effects: it changes the query param and field value. Suggest renaming, maybe something like `setTokenOnChainChange`